sdk-ios/README.md

```diff
@@ -0,0 +1,40 @@
+# DCore SDK Swift iOS Example #
+
+In this project you can find simple usage examples of [DECENT DCoreSwift SDK library - https://github.com/DECENTfoundation/DCoreSwift-SDK](https://github.com/DECENTfoundation/DCoreSwift-SDK)
+
+## List of Use Cases.
+
+Class name | Use cases
+--- | --- |
+*BalanceHistoryViewController* | Example of loading balance. Example of paginating through history.
+*CreateUIAViewController* | Example of creating and also issuing a new type of asset (UIA) into circulation.
+*CreateAccountViewController* | Example of creating account. Creating account from iOS app is however highly discouraged.
+*TransferViewController* | Transferring some amount of DCT from my account to given account name or address or id.
+*MessagesListViewController* | Reading the history of my sent or received messages.
+*MessageSendViewController* | Sending message to given account id
+*TestableViewController* | Example of amount conversion between different UIAs, created specifically to showcase how to unit test logic related to DCore SDK.
+
+## Getting Started ##
+Follow these instructions to successfuly run the iOS app on iOS Simulator.
+
+### Prerequisites ###
```

Comment:
Please add info that SWIFT 5+ is required.

Reply:
added